src/picongpu/include/plugins/common/particlePatches.cpp

```diff
@@ -18,7 +18,7 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "include/plugins/common/particlePatches.hpp"
+#include "particlePatches.hpp"
```

Comment:
please use relative path from the root `include`. In this case `plugins/common/particlePatches.hpp`